bionic/persistence.py

```diff
@@ -754,43 +750,35 @@ class LocalFilesystem:
     to local disk.
     """
 
-    def __init__(self, inventory_dir, tmp_dir):
+    def __init__(self, inventory_dir):
         self.root_url = url_from_path(inventory_dir)
-        self.tmp_root_path = tmp_dir
+        self._fs = fsspec.filesystem("file")
 
     def exists(self, url):
-        return path_from_url(url).exists()
+        return self._fs.exists(unquote(url))
```

Comment:
I notice we're unquoting all the URLs that we pass to the filesystem, but we don't do this for the GCS filesystem. Do these two filesystems make different assumptions about their inputs?

Reply:
Yeah, gcsfs unquotes by itself. But local fs doesn't, it takes the quotes literally. So if a file has space, like "file name", `path.to_uri()` will replace the space character with `%20` and without unquoting, Bionic will treat the filename as "file%20name".